repo-updater: Integrate Gitolite with the new syncer (#3884)
sourcegraph/sourcegraph: CHANGELOG.md

### Added

- Added topQueries to the GraphQL API.

> - Admins can now turn off site alerts for patch version release updates using the `alerts.showMinorUpdates` setting. Alerts will still be shown for major version updates.

Author: ```suggestion
- Admins can now turn off site alerts for patch version release updates using the `alerts.showMinorUpdates` setting. Alerts will still be shown for major and minor version updates.
```

> - Admins can now turn off site alerts for minor and patch version updates using the `alerts.showMinorUpdates` setting. Alerts will still be shown for major version updates.

Author: ```suggestion
- Admins can now turn off site alerts for patch version release updates using the `alerts.showMinorUpdates` setting. Alerts will still be shown for major version updates.
```

> - Admins can now turn off site alerts for patch version release updates using the `alerts.showMinorUpdates` setting. Alerts will still be shown for major and minor version updates.

Author: ```suggestion
- Admins can now turn off site alerts for patch version release updates using the `alerts.showPatchUpdates` setting. Alerts will still be shown for major and minor version updates.
```
